### PR TITLE
[Core] README and minor updates

### PR DESCRIPTION
**Pull request by idantene**
Adds a more detailed README file for the repository. It is strongly based on the `unmock-js` one, with attempts to make the API calls as similar as possible.
This also meant some minor changes to the `UnmockOptions` class, so that it can be edited after initializing unmock.
Also removing the `flask` and `django` boilerplate for now, so that the next merge to master will be a bit more orderly. We can always restore these as needed.
The `pytest` will remain so that we can offer a fixture out of the box.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,238 @@
-# unmock-python
-Python client for Unmock
+# Unmock (Python SDK)
```
Comment (ksaaskil) [inline README.md:1]: Perhaps add a link to `unmock` website here?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,238 @@
...
+[![CircleCI](https://circleci.com/gh/unmock/unmock-python.svg?style=svg)](https://circleci.com/gh/unmock/unmock-python)
+
+Public API mocking for Python.
+
+Unmock can be used to test modules that perform requests to third-party APIs like Hubspot, SendGrid, Behance, and
```
Comment (ksaaskil) [inline README.md:6]: Will we have API documentation for `unmock-python` (like readthedocs) and would this belong there (as well)?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,238 @@
...
+i.e. when you are in sandbox or development mode.
+For users of the [unmock.io](https://www.unmock.io) service, this will
+help unmock better organize your mocks in its web dashboard.
+
+### Headless usage
+
+Unmock works out of the box for most APIs that it mocks and does not
+require any additional configuration. For APIs that it does not mock
+yet, or to tweak return values from the unmock service, you can consult
+the URLs printed to the command line by unmock.
```
Comment (ksaaskil) [inline README.md:111]: Hmm the section `Headless usage` is probably still there by accident or?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,238 @@
...
+await unmock({
+  // ...
+  token: "my-token",
+  // ...
+}));
+```
+
+At a certain point this becomes a bit tedious, (even if very readable),
+at which point you will want to create a credentials file. See
+[unmock.io/docs](https://wwunmock.io/docs) for more information on
```
Comment (ksaaskil) [inline README.md:217]: `wwunmock.io`?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,238 @@
...
+```js
+await unmock({
+  // ...
+  token: "my-token",
+  // ...
+}));
+```
+
+At a certain point this becomes a bit tedious, (even if very readable),
+at which point you will want to create a credentials file. See
```
Comment (ksaaskil) [inline README.md:216]: I'd assume every CI environment would use environment variable instead?

**Review by ksaaskil — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (idantene) [reply README.md:6]: I think we'd better off with [our own documentation](https://www.unmock.io/docs) and just have language-specific tabs. @mikesol made a [template](https://hackernoon.com/fenced-code-tabs-in-mkdocs-material-a7951a9f8738) for that ages ago, we could use that.

Of course we could have a readthedocs as well, no harm in that.

Will keep it here meanwhile until we make a concrete decision.

File: README.md (hunk 4 above)
Comment (idantene) [reply README.md:217]: Good catch! Also in `unmock-js`.

File: README.md (hunk 5 above)
Comment (idantene) [reply README.md:216]: Hm, yeah of course you can replace that with an environment variable.
I think the point was that in every test file you'd end up writing the `token=process.env.UNMOCK_TOKEN` or similar (most of this is copied straight from the `unmock-js` readme, so I'm unsure as well :smile: )

File: README.md (hunk 2 above)
Comment (Jasone437) [reply README.md:6]: Good

**Pull request merged**